src/benchmarks/micro/corefx/System.Diagnostics/Perf_Process.cs

```diff
@@ -46,41 +39,5 @@ public void GetProcessesByName()
                 process.Dispose();
             }
         }
-        
-        [Benchmark]
-        public void StartAndWaitForExitDotNetVersion()
```

Comment:
@jorive @billwert again I am the person who created this benchmark... So the goal here was to have something that mimics a simple "Hello World" process startup time benchmark. But it turned out to be an unstable benchmark.

Since we are soon going to introduce Scenarios with proper process startup time benchmarks I am going to remove this benchmark now